webapp/src/components/kanban/kanban.tsx

```diff
@@ -278,6 +173,115 @@ class Kanban extends React.Component<Props, State> {
             await mutator.changeViewCardOrder(activeView, cardOrder, description)
         })
     }
+
+    return (
+        <div className='Kanban'>
+            <div
+                className='octo-board-header'
+                id='mainBoardHeader'
+            >
+                {/* Column headers */}
+
+                {visibleGroups.map((group) => (
+                    <KanbanColumnHeader
+                        key={group.option.id}
+                        group={group}
+                        boardTree={boardTree}
+                        intl={props.intl}
+                        addCard={props.addCard}
+                        readonly={props.readonly}
+                        propertyNameChanged={propertyNameChanged}
+                        onDropToColumn={onDropToColumn}
+                    />
+                ))}
+
+                {/* Hidden column header */}
+
+                {hiddenGroups.length > 0 &&
+                <div className='octo-board-header-cell narrow'>
+                    <FormattedMessage
+                        id='BoardComponent.hidden-columns'
+                        defaultMessage='Hidden columns'
+                    />
+                </div>
```

Comment:
Personal preference but can we indent the blocks of code within a `{}` to make it more clear that these only spawn under the condition. Otherwise I have to look for the opening and closing braces. I just find it will be more readable if we can indent these.

Reply:
It's also what's shown in the React docs https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator . Notice the indentation for multiline things within the `{}`. 
Thanks

Reply:
Done.